packages/cockpit/src/balance/operations/operations.test.js

```diff
@@ -132,6 +133,24 @@ describe('Operations table data', () => {
       expect(outgoing).toEqual(expectedOutGoing)
     })
 
+    it('should return a Bradesco transfer outgoing array', () => {
```

Comment:
talvez aqui não faça sentido se apegar a bradesco especificamente mas a credito em conta, dado que pode ocorrer com outros bancos né? Talvez
```suggestion
    it('should return a transfer outgoing array', () => {
```

Reply:
Boa, atualmente isto só ocorre pra Bradesco hoje, mas achei justo mudar o nome pra `credit transfer`.